api/idea-card/models/idea-card.settings.json

```diff
@@ -75,10 +58,39 @@
       "type": "string"
     },
     "features": {
-      "type": "string"
+      "type": "string",
+      "required": true
     },
     "experience": {
-      "type": "string"
+      "type": "string",
+      "required": true
+    },
+    "extraInfo": {
+      "type": "text"
+    },
+    "status": {
+      "type": "enumeration",
+      "enum": [
+        "submitted",
+        "applying",
+        "approved",
+        "archived",
+        "workshopping"
+      ]
+    },
+    "involveLevel": {
+      "type": "enumeration",
+      "enum": [
+        "one",
```

Comment:
Would it be easier to understand if we make this more descriptive? Like `minimum`, `medium`, `highly` etc?

Reply:
good idea. changed them to "none, minimum, medium, highly"